lib/client.js

```diff
@@ -792,62 +793,46 @@ class Client extends events.EventEmitter {
     }
 
     /**
-     * Connects and handles the execution of prepared and simple statements.
+     * Wrapper for executing queries by rust driver
      * @param {string} query
      * @param {Array} params
      * @param {ExecutionOptions} execOptions
      * @returns {Promise<ResultSet>}
      * @private
      */
-    async #execute(query, params, execOptions) {
-        const version = this.controlConnection.protocolVersion;
-
+    async #rustyExecute(query, params, execOptions) {
         if (
-            !execOptions.isPrepared() &&
+            // !execOptions.isPrepared() &&
             params &&
-            !Array.isArray(params) &&
-            !types.protocolVersion.supportsNamedParameters(version)
+            !Array.isArray(params)
+            // && !types.protocolVersion.supportsNamedParameters(version)
         ) {
-            // Only Cassandra 2.1 and above supports named parameters
-            throw new errors.ArgumentError(
-                "Named parameters for simple statements are not supported, use prepare flag",
-            );
+            throw new Error(`TODO: Implement any support for named parameters`);
+            // // Only Cassandra 2.1 and above supports named parameters
+            // throw new errors.ArgumentError(
+            //   "Named parameters for simple statements are not supported, use prepare flag",
+            // );
         }
```

Comment:
Maybe add issue for every temporally disabled feature to not forget about fixing those.